Sprint-1/exercises/random.js

```diff
@@ -7,3 +7,7 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+console.log(num);
+
+// The program creates a random whole number between 1 and 100, max and min inclusive.
```

Comment:
This exercise was expecting you to **break down**
the expression `Math.floor(Math.random() * (maximum - minimum + 1)) + minimum`, and explain
why such expression can correctly produce a random integer in the range [1, 100].

Also, to test your understanding, how would you write an expression 
(without using any variable in the expression) that yields a random 
integer between -5 and 3 (including -5 and 3)?